Contexts/MainAppContext.cs

```diff
@@ -101,6 +102,17 @@ protected override void OnModelCreating(ModelBuilder builder)
                                                        .HasForeignKey(ln => ln.LikerId)
                                                        .HasPrincipalKey(u => u.Id)
                                                        .OnDelete(DeleteBehavior.NoAction);
+            builder.Entity<Comment>()
+            .HasOne(c => c.Project)
+            .WithMany(p => p.Comments)
+            .HasForeignKey(c => c.ProjectId)
+            .OnDelete(DeleteBehavior.NoAction);
+
+            builder.Entity<Comment>()
+                .HasOne(c => c.User)
+                .WithMany()
+                .HasForeignKey(c => c.UserId)
+                .OnDelete(DeleteBehavior.Cascade);
```

Comment:
Use another relationship, Because this is different from us.

Reply:
Changing this configuration has led to some errors, so Diyar and I agreed to leave it as is currently, to focus on features delivery.

Reply:
Ok dude